examples/lfric/eg14/Makefile

```diff
@@ -31,34 +31,86 @@
 # ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 # ------------------------------------------------------------------------------
-# Author: A. R. Porter, STFC Daresbury Laboratory
-# Modified J. Henrichs, Bureau of Meteorology
+# Author: J. Henrichs, Bureau of Meteorology
+# Modifications: A. R. Porter, STFC Daresbury Laboratory
 
-include ../../common.mk
+# The compiler to use may be specified via the F90 environment
+#
+# export F90=gfortran
```

Comment:
Probably worth changing this to specify how to build for OpenACC using nvfortran.

Reply:
Done.